### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by schae234**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Rob Schaefer
```
Comment (acabunoc) [inline LICENSE]: Whoa whoa whoa!

Comment (schae234) [reply LICENSE]: My plans are foiled!

Comment (MakerTobey) [reply LICENSE]: It's a cool feature to be able to leave comments inline with the code. I didn't know that!

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Abby!
```
Comment (acabunoc) [inline LICENSE]: Thanks for changing this to me!!

**Pull request merged**